### PR TITLE
feat/#6 Add lic14 and its unit tests

### PR DESCRIPTION
**Pull request by georgewbar**
Method lic14 was implemented together with its unit tests.

### CODE REVIEW

File: src/main/java/Decide.java [hunk 1]
```diff
@@ -300,4 +300,76 @@ boolean lic10() {
...
+        // points[j+E_PTS+1+F_PTS+1] == points[j+E_PTS+F_PTS+2], ...
+
+        boolean result = false;
+        
+        for (int i = parameters.ePts + parameters.fPts + 2; i < points.size(); i++) {
+            Point2D point0 = points.get(i - parameters.ePts - parameters.fPts - 2);
+            Point2D point1 = points.get(i - parameters.fPts - 1);
+            Point2D point2 = points.get(i);
+
+            double triangleArea = calcAngle(point0, point1, point2);
```
Comment (EdvinArdo) [inline src/main/java/Decide.java]: Assigning an angle to triangleArea is a bug, correct?

File: src/main/java/Decide.java [hunk 2]
```diff
@@ -300,4 +300,76 @@ boolean lic10() {
...
+        }
+        	
+        // INVARIANT: First part is satisfied.
+        
+        for (int i = parameters.ePts + parameters.fPts + 2; i < points.size(); i++) {
+            Point2D point0 = points.get(i - parameters.ePts - parameters.fPts - 2);
+            Point2D point1 = points.get(i - parameters.fPts - 1);
+            Point2D point2 = points.get(i);
+
+            double triangleArea = calcAngle(point0, point1, point2);
```
Comment (EdvinArdo) [inline src/main/java/Decide.java]: Another line where an angle is assigned to an area.

File: src/main/java/Decide.java (hunk 1 above)
Comment (georgewbar) [reply src/main/java/Decide.java]: Fixed.

File: src/main/java/Decide.java (hunk 2 above)
Comment (georgewbar) [reply src/main/java/Decide.java]: Fixed.

**Review by EdvinArdo — APPROVED** [review]
LIC 14 ands tests look correct

**Pull request merged**